services/generic/color_control.md

```diff
@@ -1,45 +1,104 @@
+| `sup_components` | str_array | `["red", "green", "blue"]`                                                   | List of supported color components, see definition of [`color_component`](#definitions) for list of possible values. |
+| `sup_durations`  | object    | `[{"min": 1, "max": 127, "step": 1}, {"min": 180, "max": 7620, "step": 60}]` | Supported duration steps for transition change as an array of [`transition_range`](#definitions) objects.            |
 
-Supported color components:
-- Zwave: red, green, blue, warm_w, cold_w, amber, cyan, purple
-- Zigbee: red, green, blue, temp
+> In the Z-Wave protocol, color component transition duration can only be set between 1-127 seconds or 1-127 minutes. In order to fully support the Z-Wave protocol,
+> `sup_durations` property can contain an array of ranges with a specific step.
 
-#### Notes
+## Definitions
 
-- temp - is color temperature in Mired (micro reciprocal degree). It is related to Kelvins as:
-  `temp_kelvins = 1,000,000 / temp_mireds`
-  Supported `temp` values: 1-65279 mired. Actual color temperature supported by end devices is 2700K-6500K.
+* `color_component` represents a single color component, see the table below for well-defined color components:
 
-- warm_w - is warm white light source intensity. Value range 0-255.
+| Color component | Z-Wave | Zigbee | Range   | Description                                                                                                                    |
+|-----------------|--------|--------|---------|--------------------------------------------------------------------------------------------------------------------------------|
+| `red`           | Yes    | Yes    | 0-255   | Defines intensity of red color.                                                                                                |
+| `green`         | Yes    | Yes    | 0-255   | Defines intensity of green color.                                                                                              |
+| `blue`          | Yes    | Yes    | 0-255   | Defines intensity of blue color.                                                                                               |
+| `warm_w`        | Yes    | No     | 0-255   | Is intensity of warm white light source. Mix of warm white and cold white intensity forms color temperature.                   |
+| `cold_w`        | Yes    | No     | 0-255   | Is intensity of cold white light source. Mix of warm white and cold white intensity forms color temperature.                   |
+| `temp`          | No     | Yes    | 1-65279 | Is the color temperature in Mired (micro reciprocal degree). Actual color temperature supported by end devices is 2700K-6500K. |
+| `amber`         | Yes    | No     | 0-255   | Defines intensity of amber color.                                                                                              |
```

Comment:
This should be supported as well in ZIgbee there is no reason we cannot transform RGB into another color system... Or at least we need list of supported system - another message cmd.color.get_system ...

Reply:
amber/cyan/purple are present in 6-channel lights, so this is effectively not a 24 bit gamut but 48 bit one? If Zigbee protocol supports 6-channel colors then yeah, but otherwise, not really.

Reply:
@BudzikM why cmd.color.get_system? You say it's possible that one particular device could implement both color systems?
Why not having both systems included in "color_component" service props then?

Reply:
Actually we have a case like this. Markus Philipsh Hue adapter supports at the same time RGB and kind of HSL. There will be separate discussion for it. From my PoV, color component list should be enough to infer which system device supports.

Currently on RGB is supported by the mobile app.
ZW supports only RGB or RGB+ACP
Zigbee supports now only RGB
Philips support RGB and kind of HSL.

This is going to be cleared and worked out in post mortem.